INSTALL.md

```diff
@@ -212,6 +212,17 @@ If you don't want to clone the repository, you can install Alacritty directly us
 cargo install --git https://github.com/jwilm/alacritty
 ```
 
+### Debian/Ubuntu
+
+Using `cargo deb`, you can create and install a deb file.
+
+```sh
+git clone https://github.com/jwilm/alacritty.git
+cd alacritty
```

Comment:
The `git clone` and `cd` instructions can be removed, since they're already covered in the INSTALL.md.